nixos/modules/security/auditd.nix

```
@@ -4,16 +4,260 @@
   pkgs,
   ...
 }:
+let
+  cfg = config.security.auditd;
 
+  audit_config_type =
```

See the other comment about rfc-42 settings:

```suggestion
  settingsType =
```